### ISSUE TITLE
Robinhood not authenticating

### ISSUE THREAD

**Issue by chiefsmurph** [aurbano/robinhood-node#77]
`Robinhood.auth_token()` is returning undefined

it appears Robinhood may have changed the authentication api as this request is returning 404...

```
curl -v https://api.robinhood.com/api-token-auth/ \
   -H "Accept: application/json" \
   -d "username={username}&password={password}"
```

(taken from https://github.com/sanko/Robinhood/blob/master/Authentication.md#logging-in)

**Comment by cfarthingale** [comment]
Experiencing a related issue over at Robinbot: [https://github.com/yura505/robinbot/issues/18](https://github.com/yura505/robinbot/issues/18)

**Comment by aurbano** [comment]
Just merged the PR and released in 1.5.2 so I'll close this issue now.
